Resources/views/Navbar/messages.html.twig

```diff
@@ -5,7 +5,12 @@
         <span class="label label-success">{{ total }}</span>
     </a>
     <ul class="dropdown-menu">
-        <li class="header">{{ 'You have %count% messages'|trans({'%count%':total}, 'AdminLTEBundle') }}</li>
+        {% set nb_displayed_messages = messages|length %}
```

Comment:
I don't think we should change the translation here, as this would be BC break and additionally two missing strings for all existing translation resources.

What about adding the `'%displayed%':nb_displayed_messages` parameter to the original `trans()` call and then overwrite that one translation string in your application (if it is a requirement of yours)?

Reply:
I think I'll override your template. Let's keep this for the 3.0 release

Reply:
Did you try it? I am pretty sure that it should work. 
With the recently updated translation component, `trans()` works like `transchoice()` and with %count% it should support even another translation for zero.
Sounds easier to add a translation file to your app instead of duplicating the complete template.

Reply:
Its just because it seems out of place.... I certainly works... I could update the documentation, but still it seems out of place.

Reply:
As you wish, I just thought that it might make your life easier.